src/MartenBenchmarks/BenchAgainst/StorageFeatures.cs

```diff
@@ -194,6 +194,7 @@ public IEnumerable<IFeatureSchema> AllActiveFeatures(ITenant tenant)
                     return m.ForeignKeys
                         .Where(x => x.ReferenceDocumentType != m.DocumentType)
                         .Select(keyDefinition => keyDefinition.ReferenceDocumentType)
+                        .Where(referenceDocumentType => referenceDocumentType != null)
```

Comment:
I'd suggest to place it [in the first where](https://github.com/JasperFx/marten/pull/1339/files#diff-2db68eabb5a386b9eecd111babc6e92eR195). That would be the same way as it's done [above](https://github.com/JasperFx/marten/pull/1339/files#diff-0c90b6f6e976fbf2e6c8997ccaa8cccdR281)

Reply:
👍 Got it, was copying a different location where this was already done, will update ASAP.

Reply:
Done.